docs/markdown/glossary.md

```diff
@@ -0,0 +1,256 @@
+
+A (final) transfer state whereby funds have been returned to the sender.                                                                        
+
+## Route   
+
+The path a payment has taken or a future payment could take.                                                                         
+
+## Router  
+
+An ILP router is similar to a network router on an IP network. Its primary function is to forward incoming packets between a sender and a receiver. 
+
+A router may generate revenue from spreads on currency conversion, through subscription fees, or other means.
+                            
+## Routing Table   
+
+A lookup table used by a router to decide who its outgoing receiver should be (the next hop, which can be another router or the destination receiver). 
+
+## Sender  
+
+The Interledger node that is the initiator (payer) of an Interledger payment.                                                                       
+
+## Settlement  
+
+The finalization of a payment, such that the payee takes the ownership of the transferred funds.                                               
+
+## Simple Payment Setup Protocol (SPSP)
+
+An application layer protocol for exchanging payment information - destination account and secret - over HTTPS, between a payee and a payer, to facilitate payment over Interledger. 
+
+SPSP uses the STREAM as transport layer protocol. 
+
+See also: [IL-RFC 9](https://github.com/interledger/rfcs/blob/master/0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md)
+                               
+## STREAM  
+
+A Transport Protocol designed to be used with ILPv4. Application Layer protocols can use STREAM to send money and data between endpoints.
+
+See also: [IL-RFC 29](https://github.com/interledger/rfcs/blob/master/0029-stream/0029-stream.md)                                               
+
+## Streaming Payment   
+
+An ongoing payment where small amounts of money are sent over time to pay for some ongoing service or rent. For instance, a user might pay for a streaming video with a streaming payment. The term "streaming payment" is commonly reserved for cases where the interval is short, from one second up to about a day.                                                                      
+
+## Transaction 
+
+See instead: [Payment](#payment)                                                                                                                                             
+
+## Transport Layer 
+
+The transport layer transports payment data between two parties, using the [STREAM](https://github.com/interledger/rfcs/blob/master/0029-stream/0029-stream.md) protocol. Transport Layer is the middle layer of the Interledger protocol stack, which determines the condition and encoding of the data in the ILP Packet. It also determines what details the sender and receiver need to discuss beforehand.                                                                      
```

Comment:
Could we make this a little more precise?

(e.g.) The purpose of the transport layer is to ensure money and/or data sent by the sender is delivered to the recipient. This may include: splitting big payments into many smaller packets, adjusting the rate at which packets are sent, encrypting data, and providing the necessary details for the recipient to fulfill packets